### PR TITLE
Fixed error on mojquarter

### PR DESCRIPTION
**Pull request by markrangan**

### CODE REVIEW

File: R/mojquarter.R [hunk 1]
```diff
@@ -64,7 +64,7 @@ mojquarter <- function(date, type = "c") {
 
   else if(type == "f") {
 
-    date <- lubridate::ymd(date) - months(3)
+    date <- lubridate::ceiling_date(lubridate::ymd(date), "months") - months(3) - lubridate::days(1)
```
Comment (BrydenFrancesca) [inline R/mojquarter.R:67]: This fails when using a date which is not at the end of a month. lubridate::ymd(date) %m-% months(3) works and is probably clearer to use.

**Review by BrydenFrancesca — CHANGES_REQUESTED** [review]
Suggested change to format of code, using more simple lubridate "subtract month" function.

File: R/mojquarter.R (hunk 1 above)
Comment (markrangan) [reply R/mojquarter.R:67]: I've tested the function on the last day of different months and it's worked ok. The function stops working if you try dates like 2020-02-30 or 2021-09-31. Happy to discuss about this.

**Review by BrydenFrancesca — APPROVED** [review]

File: R/mojquarter.R (hunk 1 above)
Comment (BrydenFrancesca) [reply R/mojquarter.R:67]: @markrangan cheers for that, makes sense to me...if you fancy writing a decent error message for the function when someone gives a wrong date like that, please feel free!

**Pull request merged**